### ISSUE TITLE
CMake build system builds man page in PROJECT_SOURCE_DIR (rather than  PROJECT_BINARY_DIR)

### ISSUE THREAD

**Issue by hartwork** [libexpat/libexpat#158]
Current code in `CMakeLists.txt` should be using `PROJECT_BINARY_DIR` but does not do that yet:
```
    if(BUILD_doc)
        add_custom_command(TARGET expat PRE_BUILD COMMAND "${DOCBOOK_TO_MAN}" "${PROJECT_SOURCE_DIR}/doc/xmlwf.xml" && mv "XMLWF.1" "${PROJECT_SOURCE_DIR}/doc/xmlwf.1")
        expat_install(FILES "${PROJECT_SOURCE_DIR}/doc/xmlwf.1" DESTINATION share/man/man1)
    endif()
```

**Comment by kkkunche** [comment]
hartwork, is this issue fixed ?.

**Comment by hartwork** [comment]
No, the build sill puts files into source rather than build dir, see:
```shell
cd "$(mktemp -d)"
git clone --depth 1 https://github.com/libexpat/libexpat.git
cd libexpat/expat/
mkdir build && cd build
cmake ..
make
ls -l ../doc/xmlwf.1 
```

**Comment by kkkunche** [comment]
hartwork, Could you please review this change 
https://github.com/libexpat/libexpat/pull/263

**Comment by hartwork** [comment]
Fixed by #158, closing
